docs/samples/v1beta1/sklearn/v1/sklearn-multi-datatype-model/sklearn-multi-datatype-model.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
@Suresh-Nakkeran I have uploaded the model to `gs://kfserving-examples/models/sklearn/1.0/mixedtype/model.joblib`, can you help remove the above `model.joblib` and add an inferenceservice yaml example file instead ?